ts/Stock/Indicators/NATRIndicator.ts

```diff
@@ -1,143 +0,0 @@
-/* *
- *
- *  License: www.highcharts.com/license
- *
- *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
- *
- * */
-
-'use strict';
-
-import type IndicatorValuesObject from './IndicatorValuesObject';
-import type LineSeries from '../../Series/Line/LineSeries';
-import type SMAIndicator from './SMA/SMAIndicator';
-import type {
-    SMAOptions,
-    SMAParamsOptions
-} from './SMA/SMAOptions';
-import type SMAPoint from './SMA/SMAPoint';
-import BaseSeries from '../../Core/Series/Series.js';
-
-/**
- * Internal types
- * @private
- */
-declare global {
-    namespace Highcharts {
-        class NATRIndicator extends SMAIndicator {
-            public data: Array<NATRIndicatorPoint>;
-            public getValues<TLinkedSeries extends LineSeries>(
-                series: TLinkedSeries,
-                params: NATRIndicatorParamsOptions
-            ): (IndicatorValuesObject<TLinkedSeries>|undefined);
-            public options: NATRIndicatorOptions;
-            public pointClass: typeof NATRIndicatorPoint;
-            public points: Array<NATRIndicatorPoint>;
-        }
-
-        interface NATRIndicatorParamsOptions extends SMAParamsOptions {
-            // for inheritance
-        }
-
-        class NATRIndicatorPoint extends SMAPoint {
-            public series: NATRIndicator;
-        }
-
-        interface NATRIndicatorOptions extends SMAOptions {
-            params?: NATRIndicatorParamsOptions;
-            tooltip?: TooltipOptions;
-        }
-    }
-}
-
-declare module '../../Core/Series/SeriesType' {
-    interface SeriesTypeRegistry {
-        natr: typeof Highcharts.NATRIndicator;
-    }
-}
-
-// im port './ATRIndicator.js';
-
-var ATR = BaseSeries.seriesTypes.atr;
-
-/**
- * The NATR series type.
- *
- * @private
- * @class
- * @name Highcharts.seriesTypes.natr
- *
- * @augments Highcharts.Series
- */
-BaseSeries.seriesType<typeof Highcharts.NATRIndicator>('natr', 'sma',
-    /**
-     * Normalized average true range indicator (NATR). This series requires
-     * `linkedTo` option to be set and should be loaded after the
-     * `stock/indicators/indicators.js` and `stock/indicators/atr.js`.
-     *
-     * @sample {highstock} stock/indicators/natr
-     *         NATR indicator
-     *
-     * @extends      plotOptions.atr
-     * @since        7.0.0
-     * @product      highstock
-     * @requires     stock/indicators/indicators
-     * @requires     stock/indicators/natr
-     * @optionparent plotOptions.natr
-     */
-    {
-        tooltip: {
-            valueSuffix: '%'
-        }
-    },
-    /**
-     * @lends Highcharts.Series#
-     */
-    {
-        requiredIndicators: ['atr'],
```

Comment:
`requiredIndicator` is here set in the series factory, which indicates that as a prototype property it can not be part of the class pattern (because they become instance properties there). So you need to set the value in a following `extend` call.
```
extend(NATRIndicator.prototype, {
    requiredIndicators: ['atr']
});
```

Reply:
I was not aware of it, will change it!

Thanks!